app/policies/issue_policy.rb

```diff
@@ -1,3 +1,22 @@
 # frozen_string_literal: true
 class IssuePolicy < ApplicationPolicy
+  def index?
```

Comment:
from today do not add new policies.
add in controller
before_action :require_admin_and_moderator

(I was saying about it today in the evening)